src/main/java/com/kodilla/ecommercee/domain/ProductRequest.java

```diff
@@ -0,0 +1,44 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity(name = "PRODUCT_REQUESTS")
+public class ProductRequest {
+
+    @NotNull
+    @Id
+    @GeneratedValue
+    @Column(name = "PRODUCT_REQUEST_ID", unique = true)
+    private Long id;
+
+    @NotNull
+    @Column(name = "QUANTITY")
+    private Double quantity;
+
+    @NotNull
+    @ManyToOne
+    @JoinColumn(name = "PRODUCT_ID")
+    private Product product;
+
+    @ManyToMany
```

Comment:
Z tego co rozumiem product request to konkretne zamówienie, które jedna osoba włożyła do swojego indywidualnego koszyka więc jeden request może być tylko w jednym koszyku i ewentualnie może być w jednym orderze po sfinalizowaniu zamówienia

Reply:
Zgadzam się

Reply:
Racja, skoro jest połączone z koszykiem to jest automatycznie połączone bezpośrednio z osobą. Czyli będzie relacja ManyToOne

Reply:
OneToOne*